### PR TITLE
fix(passport-steam): SteamStrategyOptions should be public

### PR DESCRIPTION
**Pull request by nordowl**
See [https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70344](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70344)

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Discussion](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70344#discussioncomment-10411454)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/passport-steam/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 54bfb90544dc62e708c8e8c78a0be0a1c1ac0b83
</p>

File: types/passport-steam/passport-steam-tests.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import passport = require("passport");
 import SteamStrategy = require("passport-steam");
 
+// Usage
```
Comment (e6nlaq) [inline types/passport-steam/passport-steam-tests.ts:4]:
```suggestion
const options: SteamStrategy.SteamStrategyOptions = {
    returnURL: "http://localhost:3100",
    realm: "http://localhost:3100/login/steam/callback",
    apiKey: "12345",
};

```

**Review by e6nlaq — APPROVED** [review]
LGTM!

**Comment by nordowl** [comment]
Ready to merge

**Pull request merged**